feature/tabsflow/home/src/main/java/com/epmedu/animeal/home/presentation/ui/map/MapView.kt

```diff
@@ -42,6 +45,7 @@ import kotlinx.coroutines.FlowPreview
 import kotlinx.coroutines.channels.BufferOverflow
 import kotlinx.coroutines.flow.MutableSharedFlow
 import kotlinx.coroutines.flow.debounce
+import com.mapbox.maps.plugin.locationcomponent.R as LocationComponentR
```

Comment:
Please copy this to our own drawable folder and import it from there, this icon could change anytime in the future

Reply:
Done